lambda/rest-endpoints/src/curation.ts

```diff
@@ -134,12 +141,54 @@ export async function findRecentLocalisation(
 			testDate,
 		);
 		if (maybeLocalisation) {
-			return maybeLocalisation;
+			//if we have been given a set of curated recipes, ensure that they are not in the returned data
+			if (curatedRecipes) {
+				const deduplicatedLocalisation: FeastAppContainer = {
+					...maybeLocalisation,
+					//filter out anything that is a recipe specifier and also exists in curatedRecipes
+					items: maybeLocalisation.items.filter(
+						(row) =>
+							// eslint-disable-next-line no-prototype-builtins -- how else to do this?
+							(row.hasOwnProperty('recipe') &&
```

Comment:
Do we need to check if it is `recipe` is it not implicit?

Reply:
No, because it can also be `subCollection` or `chef` which support different fields

Reply:
Ah I might have missed it then, I thought it is already a one level deeper where we just have only recipes.
Alright, makes sense 👍